package.json

```diff
@@ -51,7 +52,7 @@
         "@vitest/coverage-c8": "^0.25.3",
         "@vitest/ui": "^0.25.3",
         "babel-plugin-react-require": "^3.1.3",
-        "eslint": "^8.27.0",
+        "eslint": "8.10.0",
```

Comment:
And it seems you downgraded eslint?

Reply:
i followed the error prompt when i made the PR, i thought that was the reason i was getting the error

Reply:
Let's revert this.. Ideally, we shouldn't make changes to the linters/formatters in a feature branch

Reply:
Alright, will do that